package.json

```diff
@@ -86,6 +86,7 @@
     "react-transition-group": "^2.5.3",
     "reselect": "^4.1.7",
     "shallowequal": "^1.1.0",
+    "styled-components": "^6.1.13",
```

Comment:
We avoid using styled-component in checkout-js, since preference is to use sass files which merchants can override for customising. Maybe we can avoid these and use classes and sass?